plugins/ICGenJetProducer.cc

```diff
@@ -61,7 +62,8 @@ void ICGenJetProducer::produce(edm::Event& event,
     dest.set_energy(src.energy());
     dest.set_charge(src.charge());
     dest.set_flavour(0);
-    dest.set_n_constituents(src.getGenConstituents().size());
+    if(is_slimmed_) dest.set_n_constituents(src.numberOfDaughters());
+    else dest.set_n_constituents(src.getGenConstituents().size());
```

Comment:
do you know if `src.numberOfDaughters()` would give the same answer as `src.getGenConstituents()` in a normal genjet (built from reco::GenParticles)? I guess we also have a problem here that there's no way to record/request the constituent particles in miniAOD as these are built from the PackedGenParticle collection right?